packages/svelte-materialify/src/components/FluidGrid/FluidGrid.svelte

```diff
@@ -0,0 +1,47 @@
+<script>
+  let klass = '';
+  export let container = false;
+  export let item = false;
+  export let spacing = null;
+  export let direction = null;
+  export let justify = null;
+  export let alignItems = null;
+  export let xs = null;
+  export let sm = null;
+  export let md = null;
+  export let lg = null;
+  export let xl = null;
+  export let style = null;
+  export { klass as class };
+
+  function breakpointClass(name, value) {
+    if (value === true) {
+      return `${name}-auto`;
+    }
+    if (Number.isInteger(value)) {
```

Comment:
I would do this^ first and then
```
if (value) return ...
```
second

Reply:
as in the following ? 
```suggestion
  function breakpointClass(name, value) {
    if (Number.isInteger(value)) {
      return `${name}-${value}`;
    }
    if (value) {
      return `${name}-auto`;
    }
    return null;
  }
```

Reply:
Yes. Should I 
![image](https://user-images.githubusercontent.com/65456722/106652713-27abc380-6596-11eb-98dd-6f086ffff60d.png)
?

Reply:
Now your function `breakpointClass` starts 2 times 😄

Reply:
gonna fix it later on manually XD

Reply:
should be fixed